pythermalcomfort/models/phs.py

```diff
@@ -266,6 +287,11 @@ def phs(
     }
 
     if limit_inputs:
+        if model == Models.iso_7933_2004.value:
```

Comment:
this code is not needed, we do not need to introduce a new variable called standard, see comment below.